Cargo.toml

```diff
@@ -43,33 +43,33 @@ categories = [
 rust-version = "1.78"
 
 [workspace.dependencies]
-lance = { version = "=0.14.2", path = "./rust/lance" }
-lance-arrow = { version = "=0.14.2", path = "./rust/lance-arrow" }
-lance-core = { version = "=0.14.2", path = "./rust/lance-core" }
-lance-datafusion = { version = "=0.14.2", path = "./rust/lance-datafusion" }
-lance-datagen = { version = "=0.14.2", path = "./rust/lance-datagen" }
-lance-encoding = { version = "=0.14.2", path = "./rust/lance-encoding" }
-lance-encoding-datafusion = { version = "=0.14.2", path = "./rust/lance-encoding-datafusion" }
-lance-file = { version = "=0.14.2", path = "./rust/lance-file" }
-lance-index = { version = "=0.14.2", path = "./rust/lance-index" }
-lance-io = { version = "=0.14.2", path = "./rust/lance-io" }
-lance-linalg = { version = "=0.14.2", path = "./rust/lance-linalg" }
-lance-table = { version = "=0.14.2", path = "./rust/lance-table" }
-lance-test-macros = { version = "=0.14.2", path = "./rust/lance-test-macros" }
-lance-testing = { version = "=0.14.2", path = "./rust/lance-testing" }
+lance = { version = "=0.15.0", path = "./rust/lance" }
+lance-arrow = { version = "=0.15.0", path = "./rust/lance-arrow" }
+lance-core = { version = "=0.15.0", path = "./rust/lance-core" }
+lance-datafusion = { version = "=0.15.0", path = "./rust/lance-datafusion" }
+lance-datagen = { version = "=0.15.0", path = "./rust/lance-datagen" }
+lance-encoding = { version = "=0.15.0", path = "./rust/lance-encoding" }
+lance-encoding-datafusion = { version = "=0.15.0", path = "./rust/lance-encoding-datafusion" }
+lance-file = { version = "=0.15.0", path = "./rust/lance-file" }
+lance-index = { version = "=0.15.0", path = "./rust/lance-index" }
+lance-io = { version = "=0.15.0", path = "./rust/lance-io" }
+lance-linalg = { version = "=0.15.0", path = "./rust/lance-linalg" }
+lance-table = { version = "=0.15.0", path = "./rust/lance-table" }
+lance-test-macros = { version = "=0.15.0", path = "./rust/lance-test-macros" }
+lance-testing = { version = "=0.15.0", path = "./rust/lance-testing" }
 approx = "0.5.1"
 # Note that this one does not include pyarrow
-arrow = { version = "51.0.0", optional = false, features = ["prettyprint"] }
-arrow-arith = "51.0"
-arrow-array = "51.0"
-arrow-buffer = "51.0"
-arrow-cast = "51.0"
-arrow-data = "51.0"
-arrow-ipc = { version = "51.0", features = ["zstd"] }
-arrow-ord = "51.0"
-arrow-row = "51.0"
-arrow-schema = "51.0"
-arrow-select = "51.0"
+arrow = { version = "52.0", optional = false, features = ["prettyprint"] }
```

Comment:
that `FixedSizeListArray` offset overflow fix is in `52.1`, but now datafusion hasn't make a release matches that arrow version, so I will upgrade them again after datafusion makes a new release

Reply:
I think I can just make that upgrade quickly.